compliance_checker/ioos.py

```diff
@@ -358,6 +358,11 @@ class IOOSBaseSOSCheck(BaseCheck):
 
 
 class IOOSSOSGCCheck(BaseSOSGCCheck, IOOSBaseSOSCheck):
+    _cc_spec = 'ioos_sosGC'
+    _cc_spec_version = '0.1'
```

Comment:
Are you sure on the IOOS SOS spec version here?

Reply:
As far as I am aware, the versions have not changed. As these were both sub-classes of the `BaseSOSGCCheck` and `IOOSBaseSOSCheck`, which had version 0.1, I used the existing version.